lit/setup-and-operations/release-notes/v4.2.3.lit

```diff
@@ -0,0 +1,25 @@
+\use-plugin{concourse-docs}
+
+\release{2018-02-15}{4.2.3}{1.18.2}{
+  \note{fix,security}{
+    Bumped our Garden runc dependency and patches Concourse against 
+    \link{CVE 2019-5736}{https://docs.google.com/document/d/1mtAbZ5GCybfXoqwJEwF4IUmpZTywMDYYzMJLZ49GbjA/edit#heading=h.9gp0p1gwtyy}.
```

Comment:
this link requires auth - can we link to the public CVE instead? something like https://nvd.nist.gov/vuln/detail/CVE-2019-5736

Reply:
updated!